src/main/features/offer/routes/defendant-response.ts

```diff
@@ -0,0 +1,44 @@
+import * as express from 'express'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Paths } from 'offer/paths'
+import { Form } from 'forms/form'
+import { DefendantResponse } from 'offer/form/models/defendantResponse'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { OfferGuard } from 'offer/guards/offerGuard'
+
+function getOffer (claim: any): any {
+  if (!claim.settlement || !claim.settlement.partyStatements) {
+    return undefined
```

Comment:
You are returning `undefined` here, but assuming that the offer will be present on the page. Please either make an assertion here that an offer is present when this route is accessed or add logic to the page which correctly handles missing offer. I would lean towards the former.